ajax/libs/prop-types/package.json

```diff
@@ -0,0 +1,28 @@
+{
+  "name": "prop-types",
+  "version": "15.5.10",
+  "filename": "prop-types.min.js",
+  "description": "Runtime type checking for React props and similar objects.",
+  "homepage": "https://facebook.github.io/react/",
```

Comment:
@extend1994 I think we can use the URL you suggested. Don't think it's gonna change anytime soon since people will still be using it for a long time. What do you think?